app/views/users/show.html.erb

```diff
@@ -1,9 +1,17 @@
 <section class="user-section">
   <h2><%= "Name: #{@user.name}" %></h2>
+        <% if current_user.pending_friends.include?(@user) %>
```

Comment:
- [ ] Same for using helpers for encapsulating view logic. Using helpers will allow us to extract the duplicated code that we have in `show` and `index` templates.